Tasks/VsTest/task.json

```diff
@@ -46,12 +48,25 @@
     ],
     "inputs": [
         {
-            "name": "testAssembly",
```

Comment:
check the template behaviior. the values given in template will work with V1 task only.